var/spack/repos/builtin/packages/laghos/package.py

```diff
@@ -46,6 +52,11 @@ def build_targets(self):
         targets.append('TEST_MK=%s' % spec['mfem'].package.test_mk)
         targets.append('CXX=%s' % spec['mpi'].mpicxx)
 
+        if self.version >= ver('3.0'):
+            targets.append('HYPRE_DIR=%s' % spec['hypre'].prefix)
+            if '+metis' in self.spec:
```

Comment:
Shouldn't we add a metis dependency?

Reply:
We also probably want a conflict for `+metis` when `@:2`

Reply:
Made metis dependency explicit.

To my understanding, ```@:2``` still depends on metis but in ways that are much less defined